### PR TITLE
feat: skip login screen when logged in

### LINKED ISSUE #84: Option to make account primary
@pd4d10 Doesn't seem too hard. I'd just have to use a `ReorderableList`. I'll open a PR for this if it's alright with you.

### PR DESCRIPTION
**Pull request by shreyas1599**
- Closes #84 
- active tab was not set in `init()`. Fixed it.

### CODE REVIEW

File: lib/models/auth.dart [hunk 1]
```diff
@@ -631,6 +631,10 @@ class AuthModel with ChangeNotifier {
       _accounts = (json.decode(str ?? '[]') as List)
           .map((item) => Account.fromJson(item))
           .toList();
+      activeAccountIndex = prefs.getInt(StorageKeys.defaultAccount);
+      _activeTab = prefs.getInt(
+              StorageKeys.getDefaultStartTabKey(activeAccount.platform)) ??
+          0;
```
Comment (pd4d10) [inline lib/models/auth.dart:637]: How would it behave if there is no account in the list?

Comment (shreyas1599) [reply lib/models/auth.dart:637]: It opens the login screen. [This](https://github.com/git-touch/git-touch/blob/a1021465a8c5b1ca5d92063a27a1d9d077013cd7/lib/home.dart#L217-L219) handles that case.

**Comment by pd4d10** [comment]
```
flutter: 2021-02-05T10:40:36.558239	D	CodeModel.init:	 read code: null, null, null
flutter: 2021-02-05T10:40:39.327024	E	AuthModel.init:	 prefs getAccount failed
NoSuchMethodError: The getter 'platform' was called on null.
Receiver: null
Tried calling: platform
#5      main
package:git_touch/main.dart:41
#6      _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:231:25)
#7      _rootRun (dart:async/zone.dart:1190:13)
#8      _CustomZone.run (dart:async/zone.dart:1093:19)
#9      _runZoned (dart:async/zone.dart:1630:10)
#10     runZonedGuarded (dart:async/zone.dart:1618:12)
#11     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:223:5)
#12     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:301:19)
#13     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
```

There are some errors in the first launch (without any shared_preferences values). Not sure if it's introduced in this PR, or an existing bug. Could you please take a look?

**Comment by shreyas1599** [comment]
Alright. I'll take a look.

**Comment by shreyas1599** [comment]
Done. Error was in setting `_activeTab` on `init()`

**Pull request merged**